lib/langchain/tool/weather.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+module Langchain::Tool
+  class Weather < Base
+    #
+    # A weather tool that gets current, forecast, or historical weather data
```

Comment:
Would you please a link to where the API key can be obtained from? Just for convenience.

Reply:
done